docs/data/remote-examples/templates/routes.yaml

```diff
@@ -8,6 +8,7 @@ repos:
     - gatsby-strapi
     - gatsby-wordpress
     - quarkus
+    - spring
```

Comment:
This is failing since we don't have a plain `spring` template: https://github.com/platformsh-templates?q=spring&type=&language=